basic_feature_repo/repo.py

```diff
@@ -0,0 +1,24 @@
+from datetime import timedelta
+
+from feast import Entity, Feature, FeatureView, FileSource, ValueType
+
+driver_hourly_stats = FileSource(
+    path="basic_feature_repo/data/driver_stats.parquet",
+    event_timestamp_column="event_timestamp",
+    created_timestamp_column="created",
+)
+
+driver = Entity(name="driver_id", value_type=ValueType.INT64, description="driver id",)
+
+driver_hourly_stats_view = FeatureView(
+    name="driver_hourly_stats",
+    entities=["driver_id"],
+    ttl=timedelta(days=365),
```

Comment:
btw I'm pretty sure this is a countdown to failing tests a year from now :) Not sure if there's an alternative though. You could make the TTL very large for this case safely I think.

Reply:
That's ok, we can optimize in a year. I dont even think providers will be around then.